tensorflow/python/eager/function.py

```diff
@@ -2940,6 +2942,11 @@ def flat_input_signature(self):
     """Returns the flattened input signature."""
     return self._function_spec.flat_input_signature
 
+  @property
```

Comment:
agree, let us remove this method, directly use self._hashable_input_signature

Reply:
Done